pyproject.toml

```diff
@@ -103,4 +104,4 @@ branch = true
 omit = ["*/tests/*"]
 
 [tool.coverage.report]
-fail_under = 85
+fail_under = 58
```

Comment:
what a drop!

Reply:
I've added `hooks` and `hooks_lib` to the coverage report, and the `hooks/post_plan.py` has no tests (yet). I want to tackle that in the future, together with terraform HCL unit tests.